packages/common/src/components/DeactivatedAccount.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { useTranslation } from "next-i18next";
+import styled from "styled-components";
+import IconButton from "./IconButton";
+import { IconArrowRight } from "hds-react";
+import { fontBold, fontRegular } from "../common/typography";
+
+const ErrorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: var(--container-width-xl);
+  margin: 0 auto;
+  gap: 42px;
+  padding-block: 42px;
+  padding-inline: var(--spacing-s);
+  --spacing-hz: var(--spacing-s);
+  img {
+    width: 185px;
+    height: auto;
+    margin: 0 auto;
+  }
+  @media (min-width: 768px) {
+    flex-direction: row;
+    padding-inline: var(--spacing-m);
+    --spacing-hz: var(--spacing-m);
+    img {
+      display: flex;
+      margin: 0;
+      margin-inline: auto;
+      width: auto;
+      max-height: 418px;
+      max-width: 350px;
+      flex-grow: 1;
+    }
+  }
+`;
+
+const TextContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 42px;
+  justify-content: center;
+  @media (min-width: 768px) {
+    order: -1;
+  }
+`;
+
+const Heading = styled.h1`
+  margin: 0;
+  line-height: 1;
+  font-size: 40px;
+  ${fontRegular}
+`;
+
+const Body = styled.p`
+  margin: 0;
+`;
+
+const Email = styled.a`
+  margin: 8px 0 0;
```

Comment:
why pixel margins?

and why margins at all? is there a styling issue for `<p><a>`? that seems a lot more fundamental problem than just chucking random `8px` top margins to some links

if the design calls for it. then either split it into separate `<p>`, use `line-height`, or the design is wrong

Reply:
This margin doesn't have any special reason to be in pixels. It is actually just left behind code, now removed as unnecessary. It didn't have any visible effect since it was block margin on an inline element (which is probably why I didn't notice it was left behind).